specifications/api/swagger.yaml

```diff
@@ -5998,6 +6010,14 @@ x-responsesBalanceTransaction: &responsesBalanceTransaction
       application/json:
         schema: *ApiSerialisedTransaction
 
+
+x-responseLatestBlockHeader: &responseLatestBlockHeader
+  200:
+    description: OK
+    content:
+      application/json:
+        schema: *ApiBlockHeader
```

Comment:
I see no point in having this response outside of the only endpoint which uses it (not gaining any code re-use buy paying the price of an indirection)